jobs/JGDAS_GLOBAL_OCEAN_ANALYSIS_RUN

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+export STRICT="NO"
+source "${HOMEgfs}/ush/preamble.sh"
+
+#############################
+# Source relevant config files
+#############################
+export EXPDIR=${EXPDIR:-${HOMEgfs}/parm/config}
+configs="base ocnanal ocnanalrun"
+config_path=${EXPDIR:-${PACKAGEROOT}/gfs.${gfs_ver}/parm/config}
+for config in ${configs}; do
+    . "${config_path}"/config."${config}"
+    status=$?
+    [[ ${status} -ne 0 ]] && exit "${status}"
+done
+
+
+##########################################
+# Source machine runtime environment
+##########################################
+. "${HOMEgfs}"/env/"${machine}".env ocnanalrun
+status=$?
+[[ ${status} -ne 0 ]] && exit "${status}"
+
+
+##############################################
+# Obtain unique process id (pid) and make temp directory
+##############################################
+export pid=${pid:-$$}
+export outid=${outid:-"LL${job}"}
+
+export DATA=${DATA:-${DATAROOT}/${jobid:?}}
+mkdir -p "${DATA}"
+cd "${DATA}" || (echo "${DATA} does not exist. ABORT!"; exit 1)
+
+
+##############################################
+# Run setpdy and initialize PDY variables
+##############################################
+export cycle="t${cyc}z"
+setpdy.sh
+. ./PDY
+
+
+##############################################
+# Determine Job Output Name on System
+##############################################
+export pgmout="OUTPUT.${pid}"
+export pgmerr=errfile
+
+
+##############################################
+# Set variables used in the script
+##############################################
+
+export CDUMP=${CDUMP:-${RUN:-"gfs"}}
+export COMPONENT=${COMPONENT:-ocean}
+
+##############################################
+# Begin JOB SPECIFIC work
+##############################################
+
+export COMOUT=${COMOUT:-${ROTDIR}/${CDUMP}.${PDY}/${cyc}/${COMPONENT}}
+
+# Prepare soca static dir
+
+ln -s "${COMOUT}"/analysis/* .
```

Comment:
Why is this in a j-job?  @CoryMartin-NOAA

Reply:
Thanks for putting the burden of explaining what I did on @CoryMartin-NOAA ! So, Cory, what's up with that?

Reply:
@aerorahul I remember what the issue was. All the yaml's , static files, backgrounds, obs, etc ... are created/linked/copied in the "prep" step under ```${COMOUT}"/analysis/``` and the content of that directory is needed by the run step.

Reply:
@guillaumevernieres follow what I have here for the aero case:
```
export DATA=${DATA:-${DATAROOT}/aeroanl_${CDATE}}
mkdir -p "${DATA}"
cd "${DATA}" || (echo "${DATA} does not exist. ABORT!"; exit 1)
```